automation/docker/Dockerfile

```diff
@@ -66,17 +66,13 @@ RUN curl -L https://dl.google.com/android/repository/sdk-tools-linux-${ANDROID_S
     && rm sdk.zip \
     && mkdir -p /build/android-sdk/.android/ \
     && touch /build/android-sdk/.android/repositories.cfg \
-    && yes | sdkmanager --licenses \
-    && sdkmanager --verbose "platform-tools" \
```

Comment:
I didn't know that `gradle` will install these on its own now, that's pretty awesome!
However, by having them installed, we have them cached for our builds - are we sure we want to download these fresh from Google's repo each build?

Reply:
Later in the Dockerfile we invoke gradle which will perform the download. So it will be cached in the image!